includes/DokanAdminReport.php

```diff
@@ -0,0 +1,822 @@
+<?php
+/**
+ * Admin report functionality.
+ *
+ * This class is a copy of WC_Admin_Report from `WC()->plugin_path() . '/includes/admin/reports/class-wc-admin-report.php'`
+ *
+ * @package Dokan\Admin\Reports
+ */
+
+namespace WeDevs\Dokan;
+
+use DateTimeZone;
+use WeDevs\Dokan\Cache;
+use WeDevs\Dokan\Utilities\OrderUtil;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Admin Report.
+ *
+ * Extended by reports to show charts and stats in admin.
+ *
+ * @package     WooCommerce\Admin\Reports
+ * @version     2.1.0
+ */
+class DokanAdminReport {
```

Comment:
I would suggest that we change the `WooCommerce` namespace, text domain, use of the `date()` function, and class name appropriately.

Also, it would be better if we could utilize the `includes/Utilities/OrderUtil.php` class to ease table and column usage.

Reply:
Vai as discussed we are not changing the data and updating the columns by using includes/Utilities/OrderUtil.php where is posible.